pkgs/development/compilers/gcc/10/default.nix

```
@@ -159,6 +159,9 @@ stdenv.mkDerivation ({
   nativeBuildInputs = [ texinfo which gettext ]
     ++ (optional (perl != null) perl)
     ++ (optional langAda gnatboot)
+    # The builder relies on GNU sed (for instance, Darwin's `sed' fails with
+    # "-i may not be used with stdin"), and `stdenvNative' doesn't provide it.
+    ++ (optional buildPlatform.isDarwin gnused)
```

```suggestion
    ++ optional buildPlatform.isDarwin gnused
```
those braces are completely optiona like the ones above.

---

Yes but I'll keep the same style